### ISSUE TITLE
Support respin tags that don't change the ingame version

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#3847]
release-20130915 has some bad packaging bugs under linux that we really should fix, but these have no effect ingame so requiring everyone to upgrade would be bad.

I propose we support an `@X` suffix for tags which is dropped by `make version` and the packaging scripts (the generated packages would be named identically to the current versions). This would let us tag release-20130915@2 with the packaging fixes while retaining compatibility with existing installs.

**Comment by Mailaender** [comment]
Our current state with release being available prior to the news post being made due to review and waiting for the download API IDs doesn't really bring us into that state anymore. When packaging mistakes happen we just delete the unannounced release, fix it and tag again. I do a run on http://build.opensuse.org every time so we also test that now. Also `release-20130915@2` only adds complications. If stuff gets worse than just `release-20130916` is always a way out.

**Comment by pchote** [comment]
I agree that our improved / slower release process (and the feature freeze branching) makes this redundant.

**Comment by pchote** [comment]
The (expected) release cadence has now slowed to the point that this is again a problem.

We have at least two issues with the latest release that don't affect the game (for people who can actually run it), but which really can't wait ~6 months for a fix to be shipped.

For the record, these are:
* The missing `ca-certificates-mono` dependency in the deb packaging, which stops the server list from working for people on older deb/ubuntu/mono releases from being able to query servers by default.
* Github deprecating the old/insecure https methods used by Powershell, which breaks dependency fetching under windows for both OpenRA and all SDK-based mods.

**Comment by PunkPun** [comment]
Is this still relevant?
